src/parsing/parsers/generic/jison/delete/delete.test.ts

```diff
@@ -0,0 +1,65 @@
+import {ColumnSuggestion, KeywordSuggestion, parseGenericSql} from '../../../../index';
+import {describe, expect, it} from '@jest/globals';
+
+// TODO: reuse it in more places
+const SUGGEST_TABLES_VALUE = {};
+
+describe('delete statement', () => {
```

Comment:
are we sure we want a global describe here? doesn't really add much value and makes code less readable due to nesting